src/main/java/org/opensearch/performanceanalyzer/commons/stats/decisionmaker/DecisionMakerConsts.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.performanceanalyzer.commons.stats.decisionmaker;
+
+
+import com.google.gson.JsonParser;
+
+public class DecisionMakerConsts {
```

Comment:
I am wondering if it will be better to retain this in RCA repo itself. We should come back and revisit this and `ConfigOverrides`